great-idea/css/index.css

```diff
@@ -3,64 +3,305 @@
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+  margin: 0 auto;
+  width: 52%;
+}
+header {
+  display: flex;
+  width: 100%;
+  height: 40px;
+  margin-bottom: 50px;
+}
+header nav {
+  flex-direction: row;
+  top: 50%;
+  font-size: 16px;
+  line-height: 1.5;
+}
+header nav a {
+  padding: 1px 5px;
+  margin: 5px 50px 0 0;
+  text-decoration: none;
+  color: gray;
+}
+header img {
+  justify-content: flex-end;
+}
+article {
+  display: flex;
+  margin-top: 100px;
+  width: 100%;
+  height: 400px;
+  margin-bottom: 30px;
+}
+article div {
+  display: flex;
+  width: 100%;
+  justify-content: flex-start;
+}
+article h1 {
+  flex-direction: row;
+  height: 220px;
+  width: 310px;
+  align-content: center;
+  font-size: 70px;
+  text-align: center;
+  padding: 10px 20px;
+}
+div h2 {
+  border: 1px solid black;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  height: 40px;
+  width: 200px;
+  text-align: center;
+  margin-left: 20%;
+  display: inline-block;
+  position: absolute;
+  top: 45%;
+  left: 7%;
+  padding-top: 10px;
+}
+
+div img {
+  display: flex;
+  flex-direction: flex-end;
+}
+section.top-section {
+  width: 100%;
+  display: flex;
+  margin-top: 50px;
+  margin-bottom: 50px;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+p.features-paragraph {
+  align-items: flex-start;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+p.about-paragraph {
+  justify-content: flex-end;
+}
+/*section div {
+  padding-right: 10px;
+  display: inline-block;
+  margin-right: 36px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  width: 450px;
+}
+section div h3 {
+  font-weight: bold;
+  margin-bottom: 30px;
+}
+h3.about {
+  margin-left: 376px;
+}
+section div p {
+  margin-bottom: 20px;
+  width: 250px;
+  font-weight: bold;
+}
+p.features-paragraph {
+  margin-right: 60px;
+  width: 350px;
+}
+p.about-paragraph {
+  width: 350px;
+  margin-left: 85%;
+  display: inline-block;
+}*/
+div.center-img {
+  display: flex;
+  width: 100%;
+}
+img.middle-img {
```

Comment:
I recommend using justify content and align items. https://www.w3schools.com/cssref/css3_pr_align-items.asp align items is going to default to stretch so you don't need to add that property.